west.yml

```diff
@@ -179,7 +179,8 @@ manifest:
       path: modules/debug/mipi-sys-t
       groups:
         - debug
-      revision: d9da086b11cda494d85f4d8a9829f505c2d5e380
+      url: https://github.com/aasthagr/mipi-sys-t
+      revision: syst_fix_add_wint_type
```

Comment:
not the right way to do this, please refer to the docs for how this should be done.

https://docs.zephyrproject.org/latest/guides/modules.html#process-for-submitting-changes-to-existing-modules

Reply:
Sure. Updated this as per the docs.